### PR TITLE
IA pour Pacman

### PR DESCRIPTION
**Pull request by hayatbellafkih**
IA pour Pacman

### CODE REVIEW

File: src/main/java/nl/tudelft/jpacman/Launcher.java [hunk 1]
```diff
@@ -177,10 +183,21 @@ public void launch() {
...
 		pacManUI = builder.build(game);
 		pacManUI.start();
+		//IApac();
+
+	}
+
+	/**
+	 * 
+	 */
+	public void IApac() {
```
Comment (Soupline69) [inline src/main/java/nl/tudelft/jpacman/Launcher.java]: Codacy Issue found: [Method names should not start with capital letters](https://www.codacy.com/app/mikis-millemaci/jpacman-framework/file/2525539198/issues/source?bid=3256381&fileBranchId=3281826#l193)

File: src/main/java/nl/tudelft/jpacman/level/Level.java [hunk 2]
```diff
@@ -349,9 +385,121 @@ public void run() {
...
+
+			Square rightSquare = board.squareAt(getCurrentPacManPosition()
+					.getX() + 1, getCurrentPacManPosition().getY());
+
+			Square northSquare = board.squareAt(getCurrentPacManPosition()
+					.getX(), getCurrentPacManPosition().getY() - 1);
+
+			Square southSquare = board.squareAt(getCurrentPacManPosition()
+					.getX(), getCurrentPacManPosition().getY() + 1);
+			switch (currentDirection) {
```
Comment (Soupline69) [inline src/main/java/nl/tudelft/jpacman/level/Level.java]: Codacy Issue found: [A switch statement does not contain a break](https://www.codacy.com/app/mikis-millemaci/jpacman-framework/file/2525539212/issues/source?bid=3256381&fileBranchId=3281826#l433)

Comment (Soupline69) [inline src/main/java/nl/tudelft/jpacman/level/Level.java]: Codacy Issue found: [Switch statements should have a default label](https://www.codacy.com/app/mikis-millemaci/jpacman-framework/file/2525539212/issues/source?bid=3256381&fileBranchId=3281826#l433)

File: src/main/java/nl/tudelft/jpacman/Launcher.java (hunk 1 above)
Comment (Mathieu1609) [inline src/main/java/nl/tudelft/jpacman/Launcher.java]: Codacy Issue found: [Method names should not start with capital letters](https://www.codacy.com/app/mathieu-craeye/jpacman-framework_2/file/2525543392/issues/source?bid=3267846&fileBranchId=3281830#l193)

File: src/main/java/nl/tudelft/jpacman/level/Level.java (hunk 2 above)
Comment (Mathieu1609) [inline src/main/java/nl/tudelft/jpacman/level/Level.java]: Codacy Issue found: [Switch statements should have a default label](https://www.codacy.com/app/mathieu-craeye/jpacman-framework_2/file/2525543406/issues/source?bid=3267846&fileBranchId=3281830#l433)

Comment (Mathieu1609) [inline src/main/java/nl/tudelft/jpacman/level/Level.java]: Codacy Issue found: [A switch statement does not contain a break](https://www.codacy.com/app/mathieu-craeye/jpacman-framework_2/file/2525543406/issues/source?bid=3267846&fileBranchId=3281830#l433)

**Comment by avandeursen** [comment]
Please use pull requests to the public framework only for well documented changes that are useful for all JPacman-Framework users.

**Pull request closed without merging**